kustomize/base-openshift/shard/cluster-role.yaml

```diff
@@ -27,25 +37,43 @@ rules:
       - secrets
       - configmaps
     verbs:
-      - '*'
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
   - apiGroups:
       - route.openshift.io
     resources:
-      - routes/custom-host
+      - routes
     verbs:
+      - get
+      - list
+      - watch
       - create
+      - delete
+      - patch
+      - update
   - apiGroups:
       - route.openshift.io
     resources:
-      - routes
+      - routes/custom-host
     verbs:
-      - '*'
+      - create
   - apiGroups:
       - monitoring.coreos.com
```

Comment:
Do we only create the route on OCP? I think you are missing some operations

Reply:
Its looks so because of side by side comparison tools. To simplify I had pasted below the route rbac :
```
- apiGroups:
    - route.openshift.io
  resources:
    - routes
  verbs:
    - get
    - list
    - watch
    - create
    - delete
    - patch
    - update

- apiGroups:
    - route.openshift.io
  resources:
    - routes/custom-host
  verbs:
    - create
```